src/components/Modal/Modal.tsx

```diff
@@ -41,6 +41,8 @@ export interface ModalProps extends FooterProps {
   sectioned?: boolean;
   /** Increases the modal width */
   large?: boolean;
+  /** Decreases the modal width */
+  small?: boolean;
```

Comment:
I agree. I think it makes more sense to have a `width` prop rather than `small` & `large`!

We've seen similar API bloat with the `Button` component. It has > 10 modifiers and depending on which ones you apply. Who knows what you'll get and what modifiers will be active 🤷 

If we're confident moving forward with the `width` prop we can deprecate `large` now since it'll need to be deprecated for at least one major version cycle before it can be removed.

Reply:
We already have the `large` prop. I would like to see two PR's one to ship small at this moment then a breaking change to replace `small` and `large`.

Reply:
That works for me. I can put up another PR for the breaking change next. Is this one good for adding the `small` prop?